tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
And then the `email_transactional` `backref` on `project.creator` should be ... a list of nodes for which the user in question is in the `email_transactional` list?